roscala/src/test/scala/coop/rchain/rosette/TransitionSpec.scala

```diff
@@ -1036,4 +1036,73 @@ class TransitionSpec extends FlatSpec with Matchers {
     val end = VirtualMachine.executeSeq(codevec, start)
     end.ctxt.ctxt.argvec.elem(16) shouldBe Fixnum(2)
   }
+
+  "OpXmitTag" should "store the result of xmit function into the specified location for future use" in {
+
+    /**
+      * This opcode sequence will be generated by the RBL
+      * compiler with the expression: (let [[x 1]] (set! x (+ 7 5)))
+      *
+      * litvec:
+      *   0:	{LetExpr}
+      *   1:	{Template}
+      *   2:	{Loc lex[0,0]}
+      * codevec:
+      *   0:	alloc 1
+      *   1:	lit 1,arg[0]
+      *   2:	nargs 1
+      *   3:	extend 1
+      *   4:	outstanding 12,1
+      *   6:	push/alloc 2
+      *   7:	lit 1,arg[0]
+      *   8:	lit 1,arg[1]
```

Comment:
`lit 1`?

Reply:
sorry， I want to make the return result more special, I will update it.